atlp-rwanda/winners-c8-bn-fe
# Logout feature

**What does this PR do?**
1. Allow user to sign out in the application

Describe what PR do.
1. Clear token(session) in the local storage.
2. redirect to the login page.

**How should this be manually tested?**
1. Clone this repository locally in your computer.
2. to install dependencies, run `npm i`
3. to start server run `npm run dev`
4. the login in the application. and logout on left-down sidebar..
5. You can check if it is working by return back to see if it can bring you back to the dashboard(Optional).

**What are the relevant pivotal tracker stories?**
Provide PT:  Logout
